packages/nestjs-firebase-admin/lib/index.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
we should export `FirebaseAdminModule` and `FirebaseProviderType`

Reply:
Sorry, but I don't know what We have to export in a library context. Sorry for ignorance.

Reply:
It's ok :), we're just creating a module used as the library's entrypoint. Every type required to use the library should be added here